source/features/comments-time-machine-links.tsx

```diff
@@ -65,12 +65,20 @@ async function showTimemachineBar(): Promise<void | false> {
 		url.pathname = pathnameParts.join('/');
 	} else {
 		// This feature only makes sense if the URL points to a non-permalink
-		const branchSelector = await elementReady('.branch-select-menu i');
+		const branchSelector = await elementReady('[data-hotkey="w"] i');
```

Comment:
Since you're editing this, can you check whether we use this exact selector everywhere in RGH?

Reply:
We do, I checked. (Should it be its own global function?)

Reply:
> Should it be its own global function

This is one of those situations that are _too specific_ yet common. Creating a function for it won't save us once we need a _slightly different_ selector and definitely won't save us from selectors in CSS files. We just need to learn to search and replace every time a selector is changed anywhere.